easy-ui-react/src/components/Placeholder/Placeholder.stories.mdx

```diff
@@ -0,0 +1,17 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Placeholder } from "./Placeholder";
+
+<Meta title="Placeholder" component={Placeholder} />
+
+export const Template = (args) => <Placeholder {...args} />;
+
+# Placeholder
+
+With `MDX`, we can define a story for `Placeholder` right in the middle of our
+Markdown documentation.
+
+<Canvas>
+  <Story name="Default" args={{ text: "Test" }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
```

Comment:
MDX looks great, glad we can start using it

did you have a feel for how you wanted to use MDX for easy-ui?

Reply:
i don't quite have a feel for how to use it yet. i think we'll have to come up with some conventions, but it's nice that it allows for more long-form documentation, which could be beneficial. that said, it does come with some cons that i'd love to talk about with the team